packages/devtools_app/lib/src/debugger/flutter/debugger_screen.dart

```diff
@@ -22,15 +22,12 @@ import '../../ui/theme.dart';
 import 'debugger_controller.dart';
 
 class DebuggerScreen extends Screen {
-  const DebuggerScreen()
-      : super(
-          DevToolsScreenType.debugger,
-          title: 'Debugger',
-          icon: Octicons.bug,
-        );
+  const DebuggerScreen() : super(id, title: 'Debugger', icon: Octicons.bug);
+
+  static const id = 'debugger';
```

Comment:
nit: here and elsewhere it seems like we are defining the id variable more times than we need to.

If you change the constructor call to 
`const DebuggerScreen() : super(id: 'debugger', title: 'Debugger', icon: Octicons.bug);`
you can delete the const id and the override of the dogPageId member.

Reply:
removed id where we can. In some cases, we want to be able to access the static screen id for things like exporting / importing json and setting a screen id for banner messages